src/commands/schemaFetch.js

```diff
@@ -38,13 +38,13 @@ const SCHEMA_QUERY = `
   }
 `;
 
-async function fetchSchema(reindex, passedTarget) {
+async function fetchSchema(reindex, passedTarget, { force }) {
   let target = passedTarget;
   if (!target) {
     target = './ReindexSchema.json';
   }
 
-  if (await isThere(target)) {
+  if (!force && await isThere(target)) {
     throw new Error(chalk.red(`File ${target} already exists.`));
```

Comment:
Let's add an instruction here:
File ... already exists. Use --force to overwrite.

(This is how people who need this option will likely discover it)